proto/google/events/cloud/pubsub/v1/events.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
How did we create this?

I'd imagine we'd use quicktype to generate the basic schemas, then tweak them a bit, something like this:

```json
{
    "$schema": "http://json-schema.org/draft-06/schema#",
    "$ref": "#/definitions/PubSubData",
    "definitions": {
        "PubSubData": {
            "type": "object",
            "additionalProperties": false,
            "properties": {
                "message": {
                    "$ref": "#/definitions/Message"
                },
                "subscription": {
                    "type": "string"
                }
            },
            "required": [
                "message",
                "subscription"
            ],
            "title": "PubSubData"
        },
        "Message": {
            "type": "object",
            "additionalProperties": false,
            "properties": {
                "data": {
                    "type": "string"
                },
                "messageId": {
                    "type": "string"
                },
                "publishTime": {
                    "type": "string",
                    "format": "date-time"
                },
                "attributes": {
                    "$ref": "#/definitions/Attributes"
                }
            },
            "required": [
                "data",
                "messageId",
                "publishTime"
            ],
            "title": "Message"
        },
        "Attributes": {
            "type": "object",
            "additionalProperties": false,
            "required": [],
            "title": "Attributes"
        }
    }
}
```

Reply:
Yeah, it would work. We have to find a way to check the generated schema against the `.proto` files though :(